### ISSUE TITLE
(S) CSS | WEB - tidy up org listing page

### ISSUE THREAD

**Issue by vincelee888** [StreetSupport/streetsupport-web#413]
example page to replicate (on dev): `/find-help/organisation/?organisation=barnabus`

###opening times

on a location's opening times listing, each day should be listed vertically, and multiple opening times (eg 09:00 - 12:00; 13:00 - 17:00) should be listed horizontally

###errant drop shadow

remove the drop shadow that is floating about when a location does not have any services

**Comment by vincelee888** [comment]
cheers, @fatters! did you find this issue via GitHub, or via one of our meetups?

**Comment by fatters** [comment]
No worries! Found it through GitHub for Hacktoberfest. Really like the project, if you've more SCSS issues (even some larger ones), I'm away from work all week this week and would be great to contribute more to it.

**Comment by fatters** [comment]
@vincelee888 Sorry to res talking in this issue but is there a place to discuss upcoming/possible changes to the site? Am I allowed to just open a new issue or will this cause conflicts? I've noticed the social icons in the footer open in the same tab whereas I'd expect these to open in new tab (with rel="nofollow" etc). Are things like this something you would just openly expect someone to make an issue about? Is this already on a backlog? Cheers
